src/core/Slider/Slider.tsx

```diff
@@ -0,0 +1,638 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import React from 'react';
+import { useMergedRefs } from '../utils/hooks/useMergedRefs';
+import { useTheme } from '../utils/hooks/useTheme';
+import '@itwin/itwinui-css/css/slider.css';
+import { Tooltip } from '../Tooltip';
+import { Placement } from 'tippy.js';
+import { StylingProps } from '../utils/props';
+
+/** Specifies domain of allowable values. */
+export interface ValueDomain {
+  min: number;
+  max: number;
+}
+
+/** Determines which segments are shown with color.
+ * 'none' - no colored tracks are displayed.
+ * 'odd-segments'- colored tracks shown in segments 1,3,5, etc. Default if number of thumbs values are odd.
+ * 'even-segments'- colored tracks shown in segments 2,4,6, etc. Default if number of thumbs values are even.
+ * 'undefined' - temporary only here because storybook sends in string 'undefined' and not value undefined
+ */
+export type TrackDisplayMode =
+  | 'none'
+  | 'odd-segments'
+  | 'even-segments'
+  | 'undefined';
+
+/** The interaction mode. Default is 1. Possible values:
```

Comment:
I started reviewing from the top and saw these duplicate comments in SliderProps. Does the type really need to be defined separately? We normally just define them with the props:

https://github.com/iTwin/iTwinUI-react/blob/cec7085c269b2f929bb0fd4b052cf47e569a9594/src/core/Buttons/Button/Button.tsx#L21

Reply:
Prettier is doing that I am NOT explicitly doing it!

Reply:
OK, I seem to always misinterpret the point of your comments. I prefer using "type" when a definition will be used in multiple places to keep accidently changing it in one occurrence and not another. In this case I use it in Slider and Track.

Reply:
Seems you've removed the explicit type (and duplicate comment) for `ThumbMoveMode`, so this is good!